roles/default/kiali-deploy/defaults/main.yml

```diff
@@ -220,6 +220,7 @@ kiali_defaults:
       is_core: false
       namespace_selector: true
       query_scope: {}
+      query_timeout: 4
```

Comment:
This should match the timeout set in `crd-docs/cr/kiali.io_v1alpha1_kiali.yaml`.   The current hardcoded value is 4s for grpc, which is the default api we use.   For some reason we have 5s for http.  And you are looking for a longer setting for your env.  How about if we just set the default to 5s.  I don't want to change from 4s to 10s as part of this change, let's just add the configurability.

Reply:
Thanks, I don't know why we had 4s for gRPC and 5s for HTTP.  But because you were hitting timeouts I thought the higher value would be better as the default for both.   My understanding is that the values in `crd-docs/cr/kiali.io_v1alpha1_kiali.yaml` should reflect the actualy defaults in `roles/default/kiali-deploy/defaults/main.yml`.  We may have to take a look and audit the crd-docs file, but not in the PR.